src/env.ts

```diff
@@ -0,0 +1,5 @@
+const env = {
+  VITE_APP_NAME: import.meta.env.VITE_APP_NAME,
```

Comment:
https://vitejs.dev/guide/env-and-mode.html

Reply:
"import.meta" is meta data available inside module files. Since "Vite" is heavily dependent on modules, it provides environment variables as part of it.